### PR TITLE
New: WP 5.7 Compatibility

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

Trello - https://trello.com/c/raylEFaO/197-wp-57-testing

### Types of changes

- Bug fix

### How has this been tested?

- Drag and drop each module.
- Check it's not breaking anything.
- Drag and drop every block with no console error in the editor.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/tabs-child/edit.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class UAGBTabsChildEdit extends Component {
 		super( ...arguments );
 	}
 	
-	componentWillMount() {
+	UNSAFE_componentWillMount() {
```
Comment (vrundakansara) [inline src/blocks/tabs-child/edit.js:24]: @RajkiranBagal - Any idea why we are using this? `UNSAFE_componentWillMount`

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/blocks/tabs-child/edit.js (hunk 1 above)
Comment (RajkiranBagal) [reply src/blocks/tabs-child/edit.js:24]: @supriyajjadhav Can we have an alternative to this?

Comment (supriyajjadhav) [reply src/blocks/tabs-child/edit.js:24]: Checking

Comment (supriyajjadhav) [reply src/blocks/tabs-child/edit.js]: @Abhijeetbhilare  At my end, I am not getting any console warning after adding the componentWillMount function in the tabs-child block. Please check this issue and update the status.

Comment (AbhijitNage123) [reply src/blocks/tabs-child/edit.js]: @vrundakansara ma'am, warning mention that `componentWillMount has been renamed, and is not recommended for use.` 
That's why I renamed the `UNSAFE_componentWillMount`. Here is more info about it - https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html

Comment (AbhijitNage123) [reply src/blocks/tabs-child/edit.js]: Also, To remove this warning  I tried removing `componentWillMount` and put the code in `componentDidMount`. I test every functionality of Tabs it's working fine. Is there any other test case @supriyajjadhav/ @RajkiranBagal

**Review by RajkiranBagal — APPROVED** [review]

**Comment by RajkiranBagal** [comment]
@AbhijitNage123 Add changelog.

File: README.md [hunk 2]
```diff
@@ -167,6 +167,12 @@ The awesome React FontIconPicker is developed by [Alessandro Benoit](http://code
 
 ## Changelog ##
 
+### 1.22.0 ###
```
Comment (vrundakansara) [inline README.md:170]: This should be under v1.21.1

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (AbhijitNage123) [reply README.md]: Updated

**Review by vrundakansara — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 **Tags:** gutenberg, blocks, gutenberg blocks, editor, block  
 **Requires at least:** 4.7  
 **Requires PHP:** 5.6  
-**Tested up to:** 5.6  
+**Tested up to:** 5.7  
 **Stable tag:** 1.21.0
```
Comment (vrundakansara) [inline README.md:8]: Add the stable tag to `1.21.1`

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/blocks/post/styling.js [hunk 4]
```diff
@@ -237,7 +237,7 @@ function styling( props ) {
 		}
 	}
 
-	if ( true === postPagination && 'empty' === paginationMarkup ) {
```
Comment (vrundakansara) [inline src/blocks/post/styling.js:240]: Why have we removed this condition?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**